### PR TITLE
Upgrade to 0.6.2

### PR DESCRIPTION
**Pull request by saraedum**

### CODE REVIEW

**Comment by saraedum** [comment]
@conda-forge-admin, please rerender.

File: recipe/patches/Makefile.patch [hunk 1]
```diff
@@ -1,46 +1,140 @@
...
+ #OPTFLAGS    =  -DGMPRATIONAL -Wuninitialized -fno-omit-frame-pointer -O3 -mavx -msse2 -ftree-vectorizer-verbose=2 -ffast-math #-DNDEBUG
  
--CFLAGS	  = $(OPTFLAGS) $(GPROFFLAG) $(ADDITIONALINCLUDEOPTIONS) #-pedantic
+-CFLAGS	  = $(OPTFLAGS) $(GPROFFLAG) $(STACTDUMP_OPTIONS) $(ADDITIONALINCLUDEOPTIONS) -std=c++0x -g $(CDDDEFINE_PREFIX) #-pedantic
+-#CFLAGS	  = $(OPTFLAGS) $(GPROFFLAG) $(STACTDUMP_OPTIONS) $(ADDITIONALINCLUDEOPTIONS) -D_GLIBCXX_DEBUG -std=c++0x -g $(CDDDEFINE_PREFIX) #-pedantic
 -CCFLAGS	  = $(CFLAGS)
 -FFLAGS	  = $(OPTFLAGS)
-+CFLAGS_LOCAL  = $(CFLAGS) $(OPTFLAGS) $(GPROFFLAG) $(ADDITIONALINCLUDEOPTIONS) #-pedantic
-+CXXFLAGS_LOCAL= $(CXXFLAGS) $(OPTFLAGS) $(GPROFFLAG) $(ADDITIONALINCLUDEOPTIONS) #-pedantic
++CFLAGS	  := $(CFLAGS) $(OPTFLAGS) $(GPROFFLAG) $(ADDITIONALINCLUDEOPTIONS) #-pedantic
```
Comment (isuruf) [inline recipe/patches/Makefile.patch]: This is missing `std=c++0x`

Comment (saraedum) [reply recipe/patches/Makefile.patch]: I'd rather not touch this patch as it is exactly as Sage has it. I added the flag to `build.sh` instead.

Comment (isuruf) [reply recipe/patches/Makefile.patch]: Sage's patch is clearly wrong. See https://github.com/conda-forge/gfan-feedstock/pull/7/files#diff-775adbc6289afcf0d96bf2fd9de2597bR93

Comment (isuruf) [reply recipe/patches/Makefile.patch]: For now, adding the flag to build.sh is fine

**Comment by saraedum** [comment]
@isuruf: I think this is ready for merge now.

**Pull request merged**